src/chorizo-sweet-potato-enchiladas.md

```diff
@@ -0,0 +1,37 @@
+# Chorizo and Sweet Potato Enchiladas
+
+Very filling with a sweet and spicy flavour.
+
+## Ingredients
+
+- 1 large sweet potato (approx 1 pound)
+- 1 bell pepper
+- 1 jalapenos (adjust based on spice preference)
+- 2 cloves garlic
+- vegetable oil
+- 3/4 pounds chorizo (mexican chorizo is preferred)
+- 8 flour tortillas
+- 1.5 cups grated cheddar or other cheese
+- 4 green onions
+- enchilada sauce:
+	- 2 tbsp vegetable oil
+	- 2 tbsp all-purpose flour
+	- 2 tbsp chili powder
+	- 2 cups water
+	- 3 oz. tomato paste
+	- 1/2 tsp cumin
+	- 1/2 tsp garlic powder
+	- 1/4 tsp cayenne pepper
+	- salt to taste
```

Comment:
Quantify salt?

Reply:
The reason I didn't quantify salt is that it varies based on your personal tastes, the step in the recipe where I add it does say that typically 1/2 tsp to 3/4 tsp is a good amount of salt.

Reply:
Sounds good. Approving now.